web/src/components/alerts/banner.tsx

```diff
@@ -32,7 +36,8 @@ export const AlertBanner: React.FC<{
         actionClose={<AlertActionCloseButton onClose={onDelete} />}
         actionLinks={
           <React.Fragment>
-            <AlertActionLink onClick={routeChange}>{t('View alert details')}</AlertActionLink>
+            <AlertActionLink onClick={routeAlert}>{t('View alert details')}</AlertActionLink>
+            <AlertActionLink onClick={routeDashboard}>{t('View health dashboard')}</AlertActionLink>
```

Comment:
I guess you forgot to run `make i18n`, but also I'm surprised, isn't that part of the CI checks? I need to check